src/test/lib/optimizer/strategy/chunk_pruning_rule_test.cpp

```diff
@@ -480,4 +481,82 @@ TEST_F(ChunkPruningRuleTest, PredicateWithCorrelatedSubquery) {
   EXPECT_EQ(stored_table_node_1->pruned_chunk_ids(), expected_chunk_ids);
 }
 
+TEST_F(ChunkPruningRuleTest, SetPrunableSubqueryScans) {
+  const auto stored_table_node_1 = StoredTableNode::make("compressed");
+  const auto a = stored_table_node_1->get_column("a");
+  const auto stored_table_node_2 = StoredTableNode::make("long_compressed");
+  const auto x = stored_table_node_2->get_column("a");
```

Comment:
I am not really a fan of `a` and `x`. Tests are easier to read with something like `stored_table_node_2_col_a` ...

Reply:
If you say so